src/StreamrClient.js

```diff
@@ -413,6 +415,9 @@ export default class StreamrClient extends EventEmitter {
             sub.once('subscribed', () => {
                 if (sub.hasResendOptions()) {
                     this._requestResend(sub)
+                    const secondResend = setTimeout(() => this._requestResend(sub), STORAGE_DELAY)
```

Comment:
Could probably do with a comment or two to explain what this is for e.g. adapt from jira ticket